Sources/Apodini/Notifications/NotificationCenter.swift

```diff
@@ -0,0 +1,235 @@
+// swiftlint:disable first_where
+//
+//  NotificationCenter.swift
+//  
+//
+//  Created by Alexander Collins on 12.11.20.
+//
+
+import class Vapor.Application
+import struct Vapor.Abort
```

Comment:
can we use `@_implementationOnly` here?

Reply:
Unfortunately this doesn't work because I use `Application` directly which is the same problem as with `Configuration`s